.ci/docker/test/verify.sh

```diff
@@ -51,60 +51,59 @@ if [ -z "${BASE_IMAGE}" ]; then
   exit 1
 fi
 
-# Check that the base image exists
-if ! baseImageInfo="$(docker manifest inspect "${BASE_IMAGE}")"; then
-  echo >&2 "No known Docker base image ${BASE_IMAGE} exists; did you pass the right name?"
-  exit 1
+DIGEST_REGEX="BASE_DIGEST=\"(sha256\:[a-f0-9\:]+)\""
+DOCKERFILE=$(<"${BASH_SOURCE%/*}/../../../operate.Dockerfile")
+echo $DOCKERFILE
+if [[ $DOCKERFILE =~ $DIGEST_REGEX ]]; then
+    DIGEST="${BASH_REMATCH[1]}"
+    echo "Digest found: $DIGEST"
+else
+    echo >&2 "Docker image digest can not be found in the Dockerfile"
+    exit 1
 fi
 
-echo "Checking for architecture ${ARCHITECTURE}"
-digestForArchitecture=$(echo "${baseImageInfo}" | jq '.manifests[] | select(.platform.architecture == "'"${ARCHITECTURE}"'") |
-.digest' )
-
-# Removing leading and trailing quotes
-digestForArchitecture="${digestForArchitecture%\"}"
-digestForArchitecture="${digestForArchitecture#\"}"
-echo "Expected base image digest from ${BASE_IMAGE} for ${ARCHITECTURE} is: ${digestForArchitecture}"
-
 imageName="${1}"
-# Iterate through all the images passed as parameter
-for imageName in "$@"
-do
-    echo "Comparing image label values for ${imageName}..."
-    # Check that the image exists
-    if ! imageInfo="$(docker inspect "${imageName}")"; then
-      echo >&2 "No known Docker image ${imageName} exists; did you pass the right name?"
-      exit 1
-    fi
+arch="${2:-amd64}"
```

Comment:
Your version of the script uses the `$ARCHITECTURE` environment variable instead of `$2` (2nd input parameter in shell scripts, what Zeebe does) in the CI so I think you have to adjust this either here on in CI:

* https://github.com/camunda/zeebe/blob/8cab53ce1582abcbccab6ef461b6e60521a3edcd/.github/workflows/operate-release-reusable.yml#L251-L252
* https://github.com/camunda/zeebe/blob/8cab53ce1582abcbccab6ef461b6e60521a3edcd/.github/workflows/operate-release-reusable.yml#L265-L266
* maybe more places

Reply:
Good catch. I fixed it here for consistency. script now uses ARCHITECTURE envvar to avoid breaking too many places in CI.